datasets/util/coco/coco_calibration_files.txt

```diff
@@ -0,0 +1,500 @@
+000000391895.jpg
```

Comment:
What is the license of this file, and where did it come from? That should be documented somewhere.